packages/commands/src/utils.mjs

```diff
@@ -11,6 +11,44 @@ import {
 } from '@audius/sdk-legacy/dist/libs.js'
 import { PublicKey } from '@solana/web3.js'
 
+let audiusSdk
+export const initializeAudiusSdk = async ({
+  apiKey = undefined,
+  apiSecret = undefined
+} = {}) => {
+  const solanaRelay = new SolanaRelay(
+    new Configuration({
+      basePath: '/solana',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      middleware: [
+        {
+          pre: async (context) => {
+            const endpoint = 'http://audius-protocol-discovery-provider-1'
+            const url = `${endpoint}${context.url}`
```

Comment:
with `context.url` always be prefixed with a forward slash? do we need one here?